source/docs/software/actuators/overview.rst

```diff
@@ -36,13 +38,20 @@ Motor controllers come in lots of shapes, sizes and feature sets. This is the fu
 
 Pneumatics
 ----------
+
 Pneumatics are a quick and easy way to make something that's in one state or another using compressed air. For information on operating pneumatics, see :doc:`pneumatics`.
 
 FRC Legal Pneumatics controllers
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
 - Pneumatics Control Module (P/N: am-2858, 217-4243)
 
+Relays
+------
+
+A relay controls power to a motor or custom electronics in an On/Off fashion.
```

Comment:
How do you control a relay in code? Perhaps this deserves an article?

Reply:
I think the relay example covers it well.  I don't think it warrants its own page.  Even if it did it wouldn't be this page right?

https://github.com/wpilibsuite/allwpilib/tree/master/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/relay

Reply:
I do think it deserves an article, but it definitely doesn't belong here (it would close #492). Specifically, the difference between kOn and kForward often confuses people. It's not intuitive that kOn makes a motor not turn.